phys2bids/bids.py

```diff
@@ -170,3 +172,58 @@ def use_heuristic(heur_file, sub, ses, filename, outdir, record_label=''):
     heurpath = os.path.join(fldr, f'{name}physio')
 
     return heurpath
+
+
+def participants_file(indir, outdir, yml, sub):
+    """[summary]
+
+    Args:
+        indir ([type]): [description]
+        outdir ([type]): [description]
+        yml ([type]): [description]
+        sub ([type]): [description]
+    """
+
+    file_path = os.path.join(outdir, 'participants.tsv')
+    if not os.path.exists(file_path):
+        # Read yaml info if file exists
+        if os.path.exists(os.path.join(indir, yml)):
+            with open(os.path.join(indir, yml)) as f:
+                yaml_data = yaml.load(f, Loader=yaml.FullLoader)
+            p_id = yaml_data['participant']['participant_id']
+            p_age = yaml_data['participant']['age']
+            p_sex = yaml_data['participant']['sex']
+            p_handedness = yaml_data['participant']['handedness']
+        else:
+            # Fill in with data from phys2bids
```

Comment:
I think a warning here is appropiate since maybe the user is not aware of the ``participants.yml``